eks-anywhere-common/Addons/Partner/Pulumi/pulumi-operator-helmrelease.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: pulumi-kubernetes-operator
+  namespace: pulumi
+spec:
+  interval: 10m
+  targetNamespace: pulumi
+  chart:
+    spec:
+      chart: pulumi-kubernetes-operator
```

Comment:
Should we specify a chart version, e.g. `0.3.0`?

Reply:
yes thats mandatory, nice catch

Reply:
Did you test the helm release did it work?

Reply:
I fixed this to use `0.3.*`, does that seem right @elamaran11 ?

Reply:
Where is the change? I dont see the version.